tests/utils/iotrace.py

```diff
@@ -463,7 +463,7 @@ def get_trace_statistics(trace_path: str, shortcut: bool = False) -> list:
         if output.stdout == "":
             raise CmdException("Invalid IO statistics", output)
 
-        return parse_json(output.stdout)
+        return parse_json(output.stdout)[0]['statistics'][0]
```

Comment:
I think, here we should return entire set, and then in test take first entry, if more entry fail test.

Reply:
In `tests/utils/dut.py` is defined `dut_cleanup()` method - it stops all running tracing processes and removes all existing traces. So I believe this check would be redundant and returning entire set would be confusing.

Reply:
Please, check if this change doesn't destroy any tests.

Reply:
`get_trace_statistics()` will have new optional parameter - this way compatibility is keept